pkg/health/health_test.go

```diff
@@ -541,6 +541,13 @@ func TestArgoApplication(t *testing.T) {
 }
 
 func TestFluxResources(t *testing.T) {
+	assertAppHealthMsg(
+		t,
+		"./testdata/flux-helmrelease-install-failed.yaml",
```

Comment:
This can be implemented by moving the example into testdata/Kubernetes/HelmRelease and add ing
```
  annotations:
    expected-ready: true
    expected-status: InstallFailed
    expected-health: unhealthy
```